### PR TITLE
Create /mnt/plumb in case it does not exist

### PR DESCRIPTION
**Pull request by dmgolubovsky**
Plumber fails, and rio does not start if it does not exist.

Signed-off-by: golubovsky <golubovsky@gmail.com>

### CODE REVIEW

**Comment by floren** [comment]
Don't we have that cool file server that creates whatever directory you
want under /mnt?

**Comment by hdonnay** [comment]
Yeah, I think we should modify the script to use `slashn` on /mnt

**Comment by elbing** [comment]
I thought on this, but if it never was in profile, I guessed it shouldn't be.
If someone tweak slashn or whatever for this, this PR and mine will be abandoned. But think that for now Harvey doesn't boot without any of two. So think quickly.

**Comment by rminnich** [comment]
/mnt is where things go that are more system level. /n is other stuff and where slashn server runs.

The problem with this change to profile is you are assuming the nodes you cpu into will let you do the mkdir, i.e. that /mnt is writeable to you. That's not really a good assumption. In fact it's almost certainly wrong because /mnt is more of a system level thing like /usr or /lib.

You can run a server that lets you write mkdir /mnt/plumb but that's just more overhead to create a directory that should already exist.

Let's just go ahead and make sure /mnt/plumb is in our repo by adding it. 

-2

File: usr/harvey/lib/profile [hunk 1]
```diff
@@ -8,6 +8,7 @@ fn cd { builtin cd $* && awd }  # for acme
 
 switch($service){
 case terminal
+	test ! -d /mnt/plumb && mkdir /mnt/plumb
```
Comment (rminnich) [inline usr/harvey/lib/profile]: Let's keep it simple. /mnt/plumb should be there, so in a later PR I put it there.

**Comment by elbing** [comment]
Fixed in #301

**Comment by dmgolubovsky** [comment]
OK, whatever is the right solution, mine was just a quick fix.

Thanks

**Comment by elbing** [comment]
And always welcome. Thanks to you.

**Pull request closed without merging**